deploy-scripts/arbitrum-test/4-blindex-updater.ts

```diff
@@ -0,0 +1,26 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import type { DeployFunction } from "hardhat-deploy/types";
+import * as constants from "../../utils/Constants";
+import { getDeployer } from "../../utils/DeployedContractsHelpers";
+import type { DeployResult } from "hardhat-deploy/dist/types";
+import { deployContract } from "../../utils/DeploymentHelpers";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  return await deployContract("BlindexUpdater", async () => {
+    const deployer = await getDeployer(hre);
+
+    const updater: DeployResult = await hre.deployments.deploy("BlindexUpdater", {
+      from: deployer.address,
+      contract: "BlindexUpdater",
+      args: [constants.botAddress[hre.network.name]]
+    });
+
+    console.log("Deployed BlindexUpdater to: " + updater.address);
+  });
+};
+
+func.id = __filename;
+func.tags = ["BlindexUpdater"];
+func.dependencies = ["StakingRewardsDistribution"];
```

Comment:
this script is not really dependent on `StakingRewardsDistribution`, I think we can remove it

Reply:
Wouldn't that cause different order between runs when I'll add another script? And then a change in addresses...

Reply:
I think the default is the a-z order, but even if this deployment script will run before the others, it shouldn't be a problem

Reply:
but anyways that's a NIT - not worth the time investing on that